test/albumTest.js

```diff
@@ -0,0 +1,69 @@
+const chai = require('chai'),
+  chaiHttp = require('chai-http'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  expect = chai.expect,
+  should = chai.should(),
+  nock = require('nock'),
+  sessionManager = require('./../app/services/sessionManager'),
+  User = require('./../app/models').User;
+
+chai.use(chaiHttp);
+
+const successfullLogin = cb => {
+  return chai
+    .request(server)
+    .post('/users/session')
+    .send({ email: 'franco.coronel@wolox.com.ar', password: 'passwordFC' });
+};
+
+describe('/albums GET', () => {
+  let request;
+  beforeEach(done => {
+    request = chai.request(server).get('/albums');
+    done();
+  });
+
+  it(`should fail because ${sessionManager.HEADER_NAME} header is not being sent`, done => {
+    User.create({
+      name: 'Franco',
+      surname: 'Coronel',
+      email: 'franco.coronel@wolox.com.ar',
+      password: 'passwordFC'
+    });
+    request.catch(err => err.should.have.status(401));
```

Comment:
Same error as the previous PR you don't need a user in this test, so you could avoid the user creation.

And also you are creating the user and making the request concurrently, so if you do this type of things you will be getting unexpected results.